package.json

```diff
@@ -86,5 +86,10 @@
     "lib/",
     "networks.json",
     "src/"
-  ]
+  ],
+  "dependencies": {
+    "@safe-global/api-kit": "^1.3.0",
+    "@safe-global/protocol-kit": "^1.2.0",
+    "@safe-global/safe-core-sdk-types": "^2.2.0"
+  }
```

Comment:
I don't think we should add additional dependencies here. AFAIU, the new dependencies are *only* used for the `tasks/` which are not included in the published NPM packages and are more intended as "developer tools". As such they should be `devDependencies`.

The reason I am being picky about this is that it affects the CoW Swap front end and would cause it to pull in additional dependencies (thus potentially increasing the final bundle size of the FE JS) that aren't ever used (since the `src/ts/` code that we export in the package does not make use of them).